moa-quickstart.md

```diff
@@ -0,0 +1,338 @@
+# MOA quickstart guide
+
+This guide walks through setting up your first MOA cluster using `moactl`.
+
+This includes the following steps:
+
+* Installing prerequisites
+* Creating your cluster
+* Adding users to your cluster
+* Cleaning up
+
+By the end of this guide you will have a Amazon Red Hat OpenShift cluster running in your AWS account.
+
+## Installing prerequisites
+
+Complete the following prerequisites before creating your MOA cluster.
+
+### Install the required binaries
+
+Install the latest version of the OpenShift CLI, also known as `oc`. View the [OpenShift documentation](https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html#installing-the-cli) for details on downloading and installing `oc` for your platform.
+
+Verify your installation by running the following command:
+
+```
+$ oc version --client
+Client Version: 4.4.0-202005231254-4a4cd75
+```
+
+Next, install the [aws-cli](https://aws.amazon.com/cli/).
+
+### Configure your AWS account
+
+Configure `aws-cli` to use the AWS account that you would like to deploy your cluster into. Modify your `~/.aws/credentials` file to specify the correct `aws_access_key_id` and `aws_secret_access_key`.
+
+```
+$ cat ~/.aws/credentials
+[default]
+aws_access_key_id = <my-aws-access-key-id>
+aws_secret_access_key = <my-aws-secret-access-key>
+```
+
+Modify your `~/.aws/config` file to specify the AWS region you want to use.
+```
+$ cat ~/.aws/config
+[default]
+output = table
+region = us-east-2
+```
+
+To verify your configuration, run the following command to query the AWS api:
+```
+$ aws ec2 describe-regions
+```
+
+### Install moactl
+
+Download and install `moactl`:
```

Comment:
We should probably link to the latest release here: https://github.com/openshift/moactl/releases/latest